tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -34,10 +34,7 @@ import (
 	"github.com/prometheus/client_golang/api"
 	"github.com/prometheus/client_golang/api/prometheus/v1"
 	"github.com/prometheus/common/model"
-
-	"istio.io/fortio/fhttp"
-
-	// flog "istio.io/fortio/log"
```

Comment:
We should remove this unused import, otherwise lint can not correctly handle this.